openmmtools/mcmc.py

```diff
@@ -378,16 +378,12 @@ def __iter__(self):
         return iter(self.move_list)
 
     def __getstate__(self):
-        serialization = dict()
-        for i, move in enumerate(self.move_list):
-            serialization['move' + str(i)] = utils.serialize(move)
-        return serialization
+        serialized_moves = [utils.serialize(move) for move in self.move_list]
```

Comment:
So this is a dict with a single key? That seems odd. Did you instead me a dict of moves?

 Why not just dictionary comprehension then:
```python
return {move: utils.serialize(move) for move in self.move_list}
```